### ISSUE TITLE
x/build: Windows Nano Server builders?

### ISSUE THREAD

**Issue by bradfitz** [golang/go#15287]
Per #15286, I guess we'll need Windows Nano Server builders at some point.

How? GCE? Azure?

Can Microsoft run them for us? Or give us Azure credit?

/cc @jstarks

**Comment by jstarks** [comment]
I've sent out some requests internally.

**Comment by bradfitz** [comment]
Thanks.

**Comment by jstarks** [comment]
Still working on this...

**Comment by bradfitz** [comment]
Any update, @jstarks?

**Comment by bradfitz** [comment]
Ping, @jstarks. Is this available for purchase yet? Or can we run it on Azure?

**Comment by alexbrainman** [comment]
Just found this https://blogs.technet.microsoft.com/nanoserver/2016/05/27/nano-server-tp5-iaas-image-in-azure-updated/ in my inbox.

Alex

**Comment by quentinmit** [comment]
Supposedly GCE is working on Nano Server support as well: https://cloudplatform.googleblog.com/2015/07/Bringing-Windows-to-Google-Cloud-Platform.html

**Comment by bradfitz** [comment]
I have a contact at Google who can help me with Windows VM things on GCE now.

**Comment by johnsonj** [comment]
We can run Server 2016 VMs on GCE which can [run](https://cloud.google.com/compute/docs/containers/#docker_on_windows) the [nano server](https://hub.docker.com/r/microsoft/nanoserver/) container.

**Comment by bradfitz** [comment]
@johnsonj, any interest in / time for setting up a Server 2016 VM base image for us?

We currently only have 2008, using the following setup instructions: https://github.com/golang/build/tree/master/env/windows

**Comment by johnsonj** [comment]
Yeah! But I really want to make a terraform to do the manual steps. I can timebox that this week and if that doesn't pan out build the image by hand by the start of next week.

**Comment by bradfitz** [comment]
> Yeah! But I really want to make a terraform to do the manual steps.

That would be awesome. Windows is Go's only builder type set up by hand.
